README.md

```diff
@@ -285,6 +285,19 @@ Note: The custom SSO CLI should have the following interface:
 $ /usr/bin/sso me@example.com scope1 scope2
 ```
 
+### --uat
+
+If true, exchanges the fetched OAuth access token with an UAT token using
```

Comment:
Not necessary oauth access token, can be self-signed access token as well.

Reply:
Removed "OAuth" - however, note that STS doesn't support self-signed JWT as of today.